player/public/js/ekstepRendererApi.js

```diff
@@ -898,6 +898,9 @@ window.EkstepRendererAPI = {
 				if (errorStack) {
 					data.err = ((errorStack.status && errorStack.status.toString()) || errorStack.message) || "-1" // Status Code(By default is -1)
 					data.stacktrace = errorStack.stack || errorStack.responseText
+					if (typeof (data.stacktrace) === "string") {
+						data.stacktrace = data.stacktrace.substring(0, 200)
```

Comment:
Change the count to 500 or 800. Check at least 3/4 lines of stack trace has to log in telemetry event. So that we can analyze with telemetry error event stack trace.